packages/main/src/Menu.ts

```diff
@@ -416,18 +365,11 @@ class Menu extends UI5Element {
 	 * @public
 	 */
 	close(): void {
-		if (this._popover) {
-			if (isPhone()) {
-				this._parentItemsStack = [];
-			}
-			this._popover.close();
-			this._popover.resetFocus();
-		}
+		this._popover?.close(false, false, true);
 	}
 
-	async _createPopover() {
-		const staticAreaItemDomRef = await this.getStaticAreaItemDomRef();
-		this._popover = staticAreaItemDomRef!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
+	_createPopover() {
+		this._popover = this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
```

Comment:
Should we always look for it or we can just keep the result  in the propety and look for it if it's empty?

Reply:
You're right. We don't have to perform redundant DOM operations here.
don